apps/src/lib/levelbuilder/CourseOfferingEditor.jsx

```diff
@@ -52,6 +58,18 @@ export default function CourseOfferingEditor(props) {
       });
   };
 
+  // Converts selected grade levels into a string for the table
+  const handleGradeLevels = e => {
+    var options = e.target.options;
```

Comment:
Should `""` be filtered out? I'm worried if someone select `(None)` by accident, this would be something like `,K,1,2`

Reply:
Okay very very fun- I've filtered it out so empty string values aren't included in the selected list, and the selected area corrects to exclude the '' value. If a user wants to select None, it's still filtered out, but array.join() on an empty array just produces an empty string. Is that okay to have in our database?

Reply:
(I added a video of this in the description so you can see what happens when users include it in their select)

Reply:
An empty string in our database (instead of a NULL value) seems totally fine to me.

The fact that it corrects like it does in the video isn't the _best_ user experience but I think that's also fine. Hopefully no one selects "None" plus other values, so this is just to protect against that.

Reply:
Okay cool. I agree that it isn't great ux/feedback wise, but the definition of 'none' is that it should never be used as part of a list, so I feel okay with it!